activesupport/lib/active_support/testing/file_fixtures.rb

```diff
@@ -0,0 +1,35 @@
+module ActiveSupport
+  module Testing
+    # Adds simple access to sample files called file fixtures.
+    # File fixtures are normal files stored in
+    # <tt>ActiveSupport::TestCase.file_fixture_directory</tt>.
+    module FileFixtures
+      extend ActiveSupport::Concern
+
+      included do
+        class_attribute :file_fixture_directory
```

Comment:
@carlosantoniodasilva I'm fine with using `file_fixture_path` for the config option but I think we need a different name for the helper then.